src/dil_demo/events.clj

```diff
@@ -115,7 +124,7 @@
 
 (defn- make-on-message-handler
   [{:keys [eori] :as config}
-   [owner-eori topic user-number :as subscription] callback-fn]
+   [owner-eori topic user-number :as subscription] handler]
```

Comment:
deze geeft een handler terug, maar neemt ook een handler als argument.... kan je dat uitleggen in de docstring?